docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -120,17 +120,17 @@ If you set these in the Field tab, then the type will apply to all fields, inclu
 
 ### Auto
 
-The Auto cell type will automatically display values with sensible defaults applied.
+The **Auto** cell type automatically displays values, with sensible defaults applied.
 
-Toggle the **Wrap text** switch, to wrap text in the cell with the longest content or the matched field in the case of field overrides.
+Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field if you've set a field override.
```

Comment:
maybe a comma? no strong feelings either way.

```suggestion
Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field, if you've set a field override.
```

Reply:
Works for me!